cashctrl_ledger/ledger.py

```diff
@@ -197,6 +203,36 @@ def settings_clear(self):
     # ----------------------------------------------------------------------
     # Accounts
 
+    def sanitize_account_groups(self, groups: pd.Series) -> pd.Series:
+        """Ensure account group paths are sanitized with a leading slash and a valid root node.
+
+        Args:
+            groups (pd.Series): A pandas Series containing account group paths.
+
+        Returns:
+            pd.Series: Sanitized account group series.
+        """
+        # Remove leading slashes
+        groups = groups.str.replace(r'^/', '', regex=True)
+        first_nodes = groups.str.replace(r'/.*', '', regex=True)
+        first_nodes = first_nodes.apply(
+            lambda g: get_close_matches(g, ROOT_ACCOUNT_NODES, cutoff=0)[0]
+        )
+        groups = groups.where(
```

Comment:
What does `groups.where( groups.isna(), ...)`do?

Reply:
There is no need to do any action for values that are NA, it looks for all but NA values

Reply:
Thanks.